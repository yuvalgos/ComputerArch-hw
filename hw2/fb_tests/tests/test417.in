w 0x7a73c
w 0x7a74c
w 0x7a7cc
r 0x7a78c
r 0x7a774
w 0x7a798
r 0x7a7b8
r 0x7a798
r 0x7a778
w 0x7a794
w 0x7a7ac
w 0x7a77c
w 0x7a780
w 0x7a768
w 0x7a7a8
r 0x7a754
r 0x7a7a0
r 0x7a754
r 0x7a768
w 0x7a79c
r 0x7a73c
w 0x7a74c
r 0x7a79c
r 0x7a764
w 0x7a754
w 0x7a7b0
w 0x7a740
r 0x7a798
r 0x7a7b0
r 0x7a758
w 0x7a788
w 0x7a734
w 0x7a7c8
w 0x7a740
w 0x7a790
w 0x7a734
r 0x7a778
w 0x7a79c
r 0x7a794
r 0x7a730
w 0x7a7cc
