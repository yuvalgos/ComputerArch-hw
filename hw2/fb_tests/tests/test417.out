L1miss=0.902 L2miss=0.973 AccTimeAvg=180.293
